doc/website/blog/2022-08-11-announcing-versioned-migration-authoring.mdx

```diff
@@ -0,0 +1,243 @@
+the desired state ("what the database should look like"), developers describe the changes themselves
+("how to reach the state"). Most of the time, this is done by creating a set of SQL files
+containing the statements needed. Each of the files is assigned a unique version and a
+description of the changes. Tools like the ones mentioned earlier are then able to
+interpret the migration files and to apply (some of) them in the correct order to
+transition to the desired database structure.
+
+The benefit of the versioned migrations approach is that it is explicit: engineers
+know _exactly_ what queries are going to be run against the database when the time
+comes to execute them.  Because changes are planned ahead of time, migration authors
+can control precisely how to reach the desired schema.  If we consider a migration as
+a plan to get from state A to state B, oftentimes multiple paths exist, each with a
+very different impact on the database. To demonstrate, consider an initial state which
+contains a table with two columns:
+```sql
+CREATE TABLE users (
+    id int,
+    name varchar(255)
+);
+```
+Suppose our desired state is:
+```sql
+CREATE TABLE users (
+    id int,
+    user_name varchar(255)
+);
+```
+There are at least two ways get from the initial to the desired state:
+* Drop the `name` column and create a new `user_name` column.
+* Alter the name of the `name` column to `user_name`.
+
+Depending on the context, either may be the desired outcome for the developer
+planning the change. With versioned migrations, engineers have the ultimate confidence
+of what change is going to happen which may not be known ahead of time in a _declarative_
+approach.
+
+### Migration Authoring
+
+The downside of the _versioned migration_ approach is, of course, that it puts the
+burden of planning the migration on developers. This requires a certain level
+of expertise that is not always available to every engineer, as we demonstrated
+in our example of setting a default value in a SQLite database above.
+
+As part of the Atlas project we advocate for a third combined approach that we call
+"Versioned Migration Authoring". Versioned Migration Authoring is an attempt to combine
+the simplicity and expressiveness of the declarative approach with the control and
+explicitness of versioned migrations.
+
+With versioned migration authoring, users still declare their desired state and use
+the Atlas engine to plan a safe migration from the existing to the new state.
+However, instead of coupling planning and execution, plans are instead written
+into normal migration files which can be checked into source control, fine-tuned manually and
+reviewed in regular code review processes.
+
+### Getting started
+
+Start by downloading the Atlas CLI. The following command will work for MacOS:
+```bash
+brew install ariga/tap/atlas
+```
+For instructions on other platforms [go here](cli/getting-started/setting-up#install-the-cli).
```

Comment:
Lets copy the tab from the install page to have instructions for the three major platforms? If I wanna follow along and have to jump somwhere else mid-read it disrupts the flow.